fv3core/decorators.py

```diff
@@ -346,25 +374,18 @@ def __init__(self, func, origin, domain, **kwargs):
             else None
         )
         self.domain = domain
-        self.func = func
-        self.stencil_object = gtscript.stencil(
-            backend=global_config.get_backend(),
-            rebuild=global_config.get_rebuild(),
-            definition=self.func,
-            **kwargs,
-        )
 
     def __call__(self, *args, **kwargs) -> None:
         if "origin" in kwargs:
-            raise ValueError("cannot pass origin to FixedOriginStencil at call time")
+            raise ValueError("Cannot pass origin to FrozenStencil at call time")
         if "domain" in kwargs:
-            raise ValueError("cannot pass domain to FixedOriginStencil at call time")
+            raise ValueError("Cannot pass domain to FrozenStencil at call time")
```

Comment:
these used to be an assert, were unintentionally changed to this, and #393 will alter... if you merge this pr first, that one can adapt, otherwise will be a merge conflict to resolve when that one goes in

Reply:
I can change these to asserts now, and then #393 will not be necessary. Does that work?

Reply:
Up to you, I don't mind either way. The tests for this take a long time, so I think would also be fine to merge this, launch the rebuild caches plan (first stopping the regression-setup and performance test that autotrigger). But if you prefer to get some more feedback before merging (since quite a big change), makes sense to incorporate it.

Reply:
I agree, could not hurt to run this one through the test suite one more time (as a big change). I think the performance gains should be worth it.